docs/SECRETS.md

```diff
@@ -0,0 +1,41 @@
+# Secrets
+
+This page describe which secrets and how they can be managed in our playbook.
```

Comment:
```suggestion
This page describe how secrets are securely managed in our playbook. It also serves as a comprehensive list of credentials in case you want to populated them manually. 
```